locales/hr/json-inline.json

```diff
@@ -1,16 +1,16 @@
 {
-    "The :attribute must be a valid role.": "This field must be a valid role.",
-    "The :attribute must be at least :length characters and contain at least one number.": "This field must be at least :length characters and contain at least one number.",
-    "The :attribute must be at least :length characters and contain at least one special character and one number.": "This field must be at least :length characters and contain at least one special character and one number.",
-    "The :attribute must be at least :length characters and contain at least one special character.": "This field must be at least :length characters and contain at least one special character.",
-    "The :attribute must be at least :length characters and contain at least one uppercase character and one number.": "This field must be at least :length characters and contain at least one uppercase character and one number.",
-    "The :attribute must be at least :length characters and contain at least one uppercase character and one special character.": "This field must be at least :length characters and contain at least one uppercase character and one special character.",
-    "The :attribute must be at least :length characters and contain at least one uppercase character, one number, and one special character.": "This field must be at least :length characters and contain at least one uppercase character, one number, and one special character.",
-    "The :attribute must be at least :length characters and contain at least one uppercase character.": "This field must be at least :length characters and contain at least one uppercase character.",
-    "The :attribute must be at least :length characters.": "This field must be at least :length characters.",
-    "The :attribute must contain at least one letter.": "This field must contain at least one letter.",
-    "The :attribute must contain at least one number.": "This field must contain at least one number.",
-    "The :attribute must contain at least one symbol.": "This field must contain at least one symbol.",
-    "The :attribute must contain at least one uppercase and one lowercase letter.": "This field must contain at least one uppercase and one lowercase letter.",
-    "The given :attribute has appeared in a data leak. Please choose a different :attribute.": "The given field has appeared in a data leak. Please choose a different field."
+    "The :attribute must be a valid role.": ":attribute mora biti valjana uloga.",
```

Comment:
Inline translations must not refer to the attribute name.

Default translation:
> "The :attribute must be a valid role.": "The :attribute field must be a valid role.",

Inline translation:
> "The :attribute must be a valid role.": "This field must be a valid role.",


https://laravel-lang.com/contributing.html#how-can-i-add-a-language-in-this-project